README.md

```diff
@@ -32,6 +34,53 @@ During your request handler you can set `request.pg.kill` to `true`, and we'll
 remove the connection from the pool instead of simply returning it to be
 reused. This is usually done when an error is detected during a query.
 
+```js
+server.route({
+    method: 'GET',
+    path: '/',
+    handler: function (req, reply) {
+
+        const cmd = 'SELECT * FROM stats;';
+
+        req.pg.client.query(cmd, (err, result) => {
+
+            if (err) {
+                request.pg.kill = true;
+
+                return reply(err);
+            }
+
+            reply('Number of rows: ', result.rows.length);
+        });
+    }
+});
+```
+
+### Outside of request handlers
+
+Outside of request handlers you can get access to the Postgres `connect`
+method. We expose it from the plugin already bound to the connection string.
+
+After plugin registration is complete you'll have access through the
+[`server.plugins` api](https://hapijs.com/api#serverplugins).
+
+```js
+const hpg = server.plugins['hapi-node-postgres'];
```

Comment:
Your plugin example would actually need to use the `server.dependency` api. https://hapijs.com/api#serverdependencydependencies-after

And the mention of `request.server.plugins['hapi-node-postgres']` being the same thing as `server.plugins['hapi-node-postgres']` isn't something I think we need to explain.

My opinion is that documenting both of these cases is the job of hapi's official documentation, not our plugin.